src/producten-openapi.yaml

```diff
@@ -680,6 +712,7 @@ components:
             uit het product type.
       required:
       - aanmaak_datum
+      - eigenaren
```

Comment:
Is this intentionally marked as required?

Reply:
Yes, a product needs at least one eigenaar.

Reply:
In that case, what happens now if you use an empty list? Or is that not allowed?

Reply:
Yes, then you will get an error, the validation method is located in the product serializer.